api/lib/resolvers.js

```diff
@@ -300,28 +301,16 @@ const transactionMetadata = async (
   }
 }
 
-const storeUser = async (_, { idToken, premium }) => {
+const registerUser = async (_, { idToken }) => {
   try {
     const decodedToken = await firebaseAdmin.auth().verifyIdToken(idToken)
-    // get user creation date
-    const userRecord = await firebaseAdmin.auth().getUser(decodedToken.uid)
     // check if user already exists in DB
     const storedUser = await database(config)('').getUser(decodedToken.uid)
-    // check if user already has premium as a custom claim
-    if (!userRecord.customClaims) {
-      // set premium field
-      await firebaseAdmin
-        .auth()
-        .setCustomUserClaims(decodedToken.uid, { premium: premium || false })
-    }
```

Comment:
we are going to need including this at some point. Why not do it now and just set it always to false?

Reply:
I mean, we need this so the firebase user object contains this piece of information, so it cna propagate and be shared both by BE and FE